Doc/library/timeit.rst

```diff
@@ -46,8 +46,17 @@ This can be achieved from the :ref:`python-interface` with::
 
 
 Note however that :mod:`timeit` will automatically determine the number of
-repetitions only when the command-line interface is used.  In the
-:ref:`timeit-examples` section you can find more advanced examples.
+repetitions only when the command-line interface is used.
+
+:mod:`timeit` also takes a callable for the statement and setup parameters::
```

Comment:
I think replacing :mod:\`timeit\` with :func:\`timeit.timeit\` would be better. Currently, it sounds like the timeit module accepts a callable.

Reply:
> for the statement and setup parameters

The name of the first parameter is actually *stmt*. Also, we mark parameter names as italic in the documentation. So, setup should be \*setup\*.

Reply:
Note that using a callable adds an overhead for calling a callable. It can be significant for simple testing code.

Reply:
```:meth:`.timeit```  is used in other places in this file I see, so I'll use that I think.

Reply:
@berkerpeksag bold. check.

Reply:
``` :meth:`.timeit` ``` refers to the Timer method inside the definition of the Timer class, while the examples above use the module-level function.

Reply:
Oh, got it. I'll fix that.

Reply:
Oh ok.